libqtile/config.py

```diff
@@ -620,10 +620,8 @@ def match_func(value):
             if _type == 'title':
                 value = client.name
             elif _type == 'wm_class':
-                value = None
-                _value = client.window.get_wm_class()
-                if _value and len(_value) > 1:
-                    value = _value[1]
+                value = client.window.get_wm_class()
+                match_func = utils.make_many(match_func, any)
```

Comment:
I think this seems simpler as `lambda values: any(match_func(val) for val in values)`, but other than that this seems very reasonable to me.

Reply:
Do you mean you preferred the first version? I changed this code because I had issues with the linting rules.

Reply:
Probably worth just fixing the linter issue, IMO. I like the lambda version better.

Reply:
Or, well, it'll be named after you fix the issue :)

Reply:
So you prefer a nested function definition ?